signup-service/src/test/java/io/mosip/signup/services/RegistrationServiceTest.java

```
@@ -1137,6 +1431,68 @@ public void register_thenAddIdentityEndpointResponseErrors_throwException() thro
         }
     }
 
+    @Test
+    public void register_thenAddIdentityEndpointResponseNullErrors_throwException() throws Exception{
```

On null errors why should we throw an exception?

---

this test case for when register response and error are null